packages/component/src/components/page-list/view/create-collection.tsx

```diff
@@ -274,9 +274,9 @@ export const EditCollection = ({
           size="large"
           data-testid="save-collection"
           type="primary"
-          disabled={isNameEmpty(value)}
+          disabled={isNameEmpty}
           onClick={() => {
```

Comment:
Could you please use useCallback here? Thx

Reply:
Done. Refactored this implementation with `useCallback`.